crates/sui-core/tests/staged/sui.yaml

```diff
@@ -1,688 +0,0 @@
----
-AbortLocation:
```

Comment:
Do not delete this file. This is critical to ensure we are not breaking compatibly